src/rohit/Exam1.java

```diff
@@ -0,0 +1,48 @@
+package rohit;
+
+import java.util.Scanner;
+
+public class Exam1 {
+
+	void display(String[] args) {
+		for (int index = 0; index < args.length; index++) {
+			System.out.println(args[index]);
+		}
+	}
+
+	String maximumlength(String[] args) {
+		String maxlength = args[0];
+		for (int index = 0; index < args.length; index++) {
+			if (args[index].length() > maxlength.length()) {
+				maxlength = args[index];
+			}
+		}
+		return maxlength;
+	}
+
+	void displayevenchar(String maxstr) {
+		for (int index = 0; index < maxstr.length(); index++) {
+			if (index % 2 == 0) {
```

Comment:
look at the requirement. we want to print even character , index starts from 0. Odd index represents even characters of the string. look at the expected output. 

input : Technocredits
output : ehordt
You should write if(index%2 != 0)